### ISSUE TITLE
New release

### ISSUE THREAD

**Issue by OrangeDog** [cavaliercoder/libzbxpgsql#136]
The last release (1.1.0) was two years ago now, and there are various unreleased changes since then.
Can we have a new release please? What's the process?

**Comment by cavaliercoder** [comment]
The process is unfortunately quite complicated for packaging and testing on so many versions of PostgreSQL/Zabbix/Linux. I am actively working on a new release now, including support for Zabbix 4 and newer Ubuntu/Debians. Unfortunately, I have very little time, but I will update this ticket once released.

**Comment by OrangeDog** [comment]
Do you need any help? Is there much more to it than adding/updating Dockerfiles in libzbxpgsql-build?

**Comment by OrangeDog** [comment]
Just to note current LTS Zabbix versions are 2.2.23, 3.0.25 and 4.0.4 - 3.2 and 3.4 are no longer supported.

**Comment by OrangeDog** [comment]
Another note: downloading the zabbix source could be automated during the build to speed things up.
The URLs are https://sourceforge.net/projects/zabbix/files/ZABBIX%20Latest%20Stable/${version}/zabbix-${version}.tar.gz

**Comment by OrangeDog** [comment]
I've made [some progress](https://github.com/OrangeDog/libzbxpgsql-build/commit/94fa592) but the build is failing:
> \+ install -m 644 query.conf /root/rpmbuild/BUILDROOT/libzbxpgsql-1.2.0-1.x86_64/etc/libzbxpgsql.d/
> install: cannot stat `query.conf': No such file or directory
> error: Bad exit status from /var/tmp/rpm-tmp.jYCXUr (%install)

**Comment by cavaliercoder** [comment]
Yes, I have this issue too. That file's name was changed at some. You're welcome to to update the name in your build (I think `libzbxpgsql.conf`)

**Comment by OrangeDog** [comment]
😕 the config files in the repo are different to what's in e.g. [libzbxpgsql_1.1.0-1+trusty_amd64.deb](http://cdn.cavaliercoder.com/libzbxpgsql/apt/zabbix32/ubuntu/trusty/amd64/libzbxpgsql_1.1.0-1%2Btrusty_amd64.deb)
Did you build packages locally but never push the changes to github?

**Comment by cavaliercoder** [comment]
Possibly 😬I'll try get a moment to fix this for you. Sorry for the pain!!!

**Comment by mjtrangoni** [comment]
Hi @cavaliercoder,
I need a new RPM build because of #139. How does the status here look like? May I contribute with you on something?

**Comment by OrangeDog** [comment]
Releases are now not available for **any** currently supported Debian system.

**Comment by OrangeDog** [comment]
@cavaliercoder please try to find a moment to fix the build, or provide enough information for someone else to try.
